### PR TITLE
feat: support pwa hybrid mode deployment

### PR DESCRIPTION
**Pull request by jometzner**
## PR Type

[ ] Bugfix
[ X ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no API changes)
[ ] Build-related changes
[ ] CI-related changes
[ ] Documentation content changes
[ ] Application / infrastructure changes

## What Is the Current Behavior?

Intershop PWA supports hybrid mode where one can gradually switch from classic storefront pages to Angular based pages. This is also called hybrid approach. A helm chart supporting that kind of deployment was yet missing.

Issue Number: Closes #

## What Is the New Behavior?

Deploy Intershop PWA in hybrid mode is now supported.

## Does this PR Introduce a Breaking Change?

[ X ] Yes
[ ] No

## Other Information
[hybrid-approach.md](https://github.com/intershop/intershop-pwa/blob/develop/docs/concepts/hybrid-approach.md)

### CODE REVIEW

**Comment by khauser** [comment]
Could you also set the icm-chart as dependency instead of icm-as and icm-web?

**Comment by jometzner** [comment]
> Could you also set the icm-chart as dependency instead of icm-as and icm-web? 

It didn't work out the last time. I can check again next time.

**Comment by khauser** [comment]
`helm repo add intershop https://intershop.github.io/helm-charts` should be in the lint test.

**Review by andreassteinmann — DISMISSED** [review]

**Review by andreassteinmann — APPROVED** [review]

File: charts/pwa/Chart.lock [hunk 1]
```diff
@@ -0,0 +1,6 @@
+dependencies:
```
[the remark this answers is not in the document]
Comment (jometzner) [reply charts/pwa/Chart.lock:1]: Why? The hash code should be stable when referring to a release in some upstream repo.

[the remark this answers is not in the document]
Comment (jometzner) [reply charts/pwa/Chart.lock:1]:
```zsh
~/work/helm-charts/charts/pwa feature/add-hybrid-chart docker-desktop/default
❯ gst 
On branch feature/add-hybrid-chart
Your branch is up to date with 'origin/feature/add-hybrid-chart'.

nothing to commit, working tree clean

~/work/helm-charts/charts/pwa feature/add-hybrid-chart docker-desktop/default
❯ helm dependency update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "intershop" chart repository
...Successfully got an update from the "nginx-stable" chart repository
...Successfully got an update from the "prometheus-community" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading icm from repo https://intershop.github.io/helm-charts
Deleting outdated charts

~/work/helm-charts/charts/pwa feature/add-hybrid-chart docker-desktop/default
❯ gst
On branch feature/add-hybrid-chart
Your branch is up to date with 'origin/feature/add-hybrid-chart'.

nothing to commit, working tree clean
```

**Review by khauser — APPROVED** [review]

**Pull request merged**